### PR TITLE
App profiles

### PR DESCRIPTION
**Pull request by ritawatson**
This PR updates App Profile-related docs with newly added Services. It also reorganizes two docs (**App Profiles** and **Create an App Profile**) to shorten the how-to and move Services descriptions to the top-level **App Profiles** doc. 

Jira: https://spectrocloud.atlassian.net/browse/PPD-792

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1025#" title="Author self-approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/04.5-devx/05-app-profile.md
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/05-app-profile.md:25]:
```suggestion
# Services
```
I think we should make this a higher level. That way, the services below, such as databases, fall under this umbrella.

File: content/docs/04.5-devx/05-app-profile/00-create-app-profile.md
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/05-app-profile/00-create-app-profile.md:17]: Nice use of linking to sibling resources 😄

Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/05-app-profile/00-create-app-profile.md:17]: We should add after this paragraph a sentence that explains what will happen next. Such as "use the following steps to create an app profile" or something along those lines.

File: content/docs/04.5-devx/05-app-profile/00-create-app-profile.md
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/05-app-profile/00-create-app-profile.md:25]: We need the enablement header pr the [How-to layout guide](https://spectrocloud.atlassian.net/wiki/spaces/DE/pages/1774944263/How-to+Layout)

As you pointed out, we don't want to use the page title again. So maybe we can do # Enablement or # Profile Creation. I'll defer to you for the proper title.

File: content/docs/04.5-devx/05-app-profile/00-create-app-profile.md
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/05-app-profile/00-create-app-profile.md:27]:
```suggestion
1. Log in to the [Palette Dev Engine console](console.spectrocloud.com).
```

File: content/docs/04.5-devx/05-app-profile/00-create-app-profile.md
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/05-app-profile/00-create-app-profile.md:84]: Why are we removing the validation steps? Every how-to needs a validation.

**Review by karl-cardenas-coding — CHANGES_REQUESTED** [review]
Left some questions and suggestions. @ritawatson in this PR, can you also fix line 29 in docs/04.5-devx/05-app-profile/02-container-deployment por favor?

File: content/docs/04.5-devx/05-app-profile/00-create-app-profile.md
Comment (ritawatson) [reply content/docs/04.5-devx/05-app-profile/00-create-app-profile.md:84]: Validation section is back in.

File: content/docs/04.5-devx/05-app-profile/00-create-app-profile.md
Comment (ritawatson) [reply content/docs/04.5-devx/05-app-profile/00-create-app-profile.md:29]: Added Enablement header.

**Review by karl-cardenas-coding — APPROVED** [review]
@ritawatson, the changes look good. Just a minor suggestion, and one information architecture question remain.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1025#pullrequestreview-1209251136" title="Approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1025#" title="Author self-approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/04.5-devx/05-app-profile.md
Comment (tiwarisumit3) [inline content/docs/04.5-devx/05-app-profile.md:86]: Add one liner about Kafka

File: content/docs/04.5-devx/05-app-profile.md
Comment (ritawatson) [reply content/docs/04.5-devx/05-app-profile.md:96]: @tiwarisumit3 We don't document upcoming features, so I'll remove mention of Amazon S3 and any others that will not be released. Could you please list the services that won't be released?

File: content/docs/04.5-devx/05-app-profile.md
Comment (ritawatson) [reply content/docs/04.5-devx/05-app-profile.md:86]: @tiwarisumit3 Instead of one-liners, I'll add links to these so users can read more if they want.

File: content/docs/04.5-devx/05-app-profile.md
Comment (tiwarisumit3) [inline content/docs/04.5-devx/05-app-profile.md:21]: For example, you may want to create an App Profile with Helm Charts, Manifests, and different database applications.

File: content/docs/04.5-devx/05-app-profile.md
Comment (tiwarisumit3) [inline content/docs/04.5-devx/05-app-profile.md:106]: Not only database services, but similar is possible for messaging, object storage etc

File: content/docs/04.5-devx/05-app-profile/00-create-app-profile.md
Comment (ritawatson) [reply content/docs/04.5-devx/05-app-profile/00-create-app-profile.md]: I tried moving this up, but it feels right to me here because it follows the user workflow. It isn't until they navigate to the App Profiles page that they can view or modify Settings (edit/delete). I made it a single paragraph, though, and slightly reworded: "From this page, you can also edit and delete App Profiles in Settings."

File: content/docs/04.5-devx/05-app-profile.md
Comment (ritawatson) [reply content/docs/04.5-devx/05-app-profile.md]: decided against links.

File: content/docs/04.5-devx/05-app-profile.md
Comment (ritawatson) [reply content/docs/04.5-devx/05-app-profile.md:21]: Done

File: content/docs/04.5-devx/05-app-profile.md
Comment (ritawatson) [reply content/docs/04.5-devx/05-app-profile.md:106]: Revised the first sentence to say "Variables used in services follow a usage hierarchy. "

**Comment by ritawatson** [comment]
These last changes are made:

CleanShot 2022-12-08 at 16 35 00

and

CleanShot 2022-12-08 at 16 36 31

**Pull request merged**
